test/controllers/articles_controller_test.rb

```diff
@@ -3,28 +3,86 @@
 require "test_helper"
 
 # Controller tests for news articles
-class ArticleControllerTest < FunctionalTestCase
-  def test_create_article_get
+class ArticlesControllerTest < FunctionalTestCase
+  ############ test Actions that Display data (index, show, etc.)
+
+  def test_index
+    # Prove any user can see article index
+    get(:index_articles)
+    assert(:success)
+
+    # Prove privileged user get link to create an article
+    login(users(:article_writer).login)
+    get(:index_articles)
+    assert_select("a", text: :create_article_title.l)
+  end
+
+  def test_list_articles
```

Comment:
`test_list_articles` will probably go away.
The aliases will be replaced by routes which re-direct, and there will be a test. It will test the routes. It might also test that redirects actually happen, but my instinct is that this is testing Rails, rather than MO.